db/migrate/20211111114547_change_integer_id_columns_type.spree_api.rb

```diff
@@ -0,0 +1,10 @@
+# This migration comes from spree_api (originally 20210727102516)
+class ChangeIntegerIdColumnsType < ActiveRecord::Migration[5.2]
```

Comment:
why we added this migration

Reply:
i havent added this migration so i have deleted it